src/mongocxx/test/spec/monitoring.cpp

```diff
@@ -125,6 +125,12 @@ void apm_checker::has(bsoncxx::array::view expectations) {
     }
 }
 
+bool apm_checker::should_ignore(stdx::string_view command_name) const {
+    return std::any_of(std::begin(_ignore), std::end(_ignore), [command_name](const auto& s) {
+        return command_name == s;
+    });
```

Comment:
> error: no matching function for call to object of type '(lambda at /src/mongocxx/test/spec/monitoring.cpp:129:64)'

```suggestion
    return std::any_of(std::begin(_ignore),
                       std::end(_ignore),
                       [&command_name](stdx::string_view elem) { return elem == command_name; });
```

`stdx::string_view` may not/does not define heterogeneous `operator==` with `std::string` objects. Also, generic lambdas are a C++14 feature (AFAIK the C++ driver still only supports C++11).

Reply:
1) right, that we're constructing a new string_view from the std::strings in _ignore as we visit them, to compare the string_views. (And this is also why writing something like any_of(_ignore, std::equal_to(command_name)) won't compile. :-/)
2) Oh, shoot... that's probably why I'd written it the way I had it before after watching it break on CI. Le sigh.